packages/cactus-plugin-odap-hermes/src/test/typescript/fabric-contracts/lock-asset/chaincode-typescript/tslint.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@jscode017 This seems like it's not being used. If the latter is correct then please delete the file.

Reply:
Sure, I would delete it

> @jscode017 `1.0.0-rc.2` just went out so you'll also need to update everything in your package from `1.0.0-rc.1` to `1.0.0-rc.2` accordingly.

sure, I have update it

Reply:
done